### ISSUE TITLE
Sort Asc/Desc

### ISSUE THREAD

**Issue by CattailNu** [CodingFriends/Tincta#60]
Feature request: Sort selected text asc/desc, with numbers handled as numbers vs text (option?).

**Comment by greimers** [comment]
Do you mean sorting selected lines alphabetically?
Can you maybe give an example?

**Comment by CattailNu** [comment]
Yes, sorting selected lines alphabetically (either ascending or descending, with an option for numbers as numbers as text.  If the following 4 lines were selected:
bca
abc
110
10
Select Edit|Sort and have it change to:
abc
bca
10
110
(and so on)

I have a workaround using Keyboard Maestro for now.

**Comment by f1nnix** [comment]
macOS native workaround: create Text Sevice via shortcut or Automator
